.vscode/settings.json

```diff
@@ -43,5 +43,6 @@
 	"workbench.editorAssociations": {
 		"*.dmi": "dmiEditor.dmiEditor"
 	},
-	"Lua.diagnostics.enable": false
+	"Lua.diagnostics.enable": false,
+	"editor.inlineSuggest.showToolbar": "onHover"
```

Comment:
Is this change to the vs settings intentional? If so, shouldn't be in its own PR?

Reply:
Whoops, this unintentional